client/components/select-vertical/suggestion-search.tsx

```diff
@@ -0,0 +1,118 @@
+import { Suggestions } from '@automattic/components';
+import classnames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import { useCallback, useMemo, useRef, useState } from 'react';
+import * as React from 'react';
```

Comment:
Why we import react twice?

Reply:
This is just to be avoid explicitly importing `ChangeEvent` and `FC`, but I can update it.